### ISSUE TITLE
[QUESTION] Missing styles

### ISSUE THREAD

**Issue by edtorba** [markmead/hyperui#207]
First of all thanks for this great UI kit.

Working with tailwindcss v3.2.1

Noticing the following input & select elements in my case are missing padding etc properties and is not browser specific.

Got following

```css
@tailwind base;
@tailwind components;
@tailwind utilities;
```

Examples:
The input elements missing padding.
<img width="664" alt="Screenshot 2022-10-25 at 21 19 38" src="https://user-images.githubusercontent.com/2422019/197877576-045bbe5c-d21f-4028-ab6d-bd96da63fbeb.png">

The select element missing pretty much most of the styling.
<img width="1616" alt="Screenshot 2022-10-25 at 21 20 03" src="https://user-images.githubusercontent.com/2422019/197877616-75464af7-d816-414b-acc5-9a84440049c0.png">

After inspecting the select element, have noticed that am missing at least the following properties.
<img width="869" alt="Screenshot 2022-10-25 at 21 43 46" src="https://user-images.githubusercontent.com/2422019/197878194-b917cdd4-a537-4e9b-ab53-9e12c18501ed.png">

TLDR; Are there any prerequisites? Apart from tailwind framework and imports that are mentioned above?

**Comment by edtorba** [comment]
Turns out that those components also rely on styles that come from `@tailwindcss/forms` plugin

**Comment by markmead** [comment]
Yeah, that's the ticket @edtorba! 

I usually leave comments saying what is needed for each component but I seem to have missed a few, I'll update soon.

**Comment by edtorba** [comment]
@markmead let me know if you need any assistance 👍

**Comment by markmead** [comment]
Thanks, @edtorba I've already done it with #208 but if you spot any missing then feel free to make an issue/PR.
